tools/virtualization/playing-around-with-various-vga-providers.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Playing around with various VGA providers  
+subsection: virtualization  
+order: 11  
+---
+
+# Playing around with various VGA providers
+
+> More information can be found here https://www.kraxel.org/blog/2019/09/display-devices-in-qemu/
+
+1. Download the Fedora Workstation ISO file from website and store it in a reference directory `cdromimg`.  
+   ```
+   https://getfedora.org
+   ```
```

Comment:
```suggestion
```
I think this should be already covered elsewhere. F.e. it can be moved to `about` page.